webapp/src/main/java/com/box/l10n/mojito/metrics/NamingConventionBuilder.java

```diff
@@ -0,0 +1,35 @@
+package com.box.l10n.mojito.metrics;
+
+import com.google.common.base.Strings;
+
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.config.NamingConvention;
+import io.micrometer.core.lang.Nullable;
+
+public class NamingConventionBuilder {
+
+    public static NamingConvention build(StatsdNamingConventionConfig config) {
+
+        NamingConvention convention = config.getNamingConvention();
+        String namePrefix;
+
+        if (Strings.isNullOrEmpty(config.getNamePrefix()) || config.getNamePrefix().endsWith(".")) {
```

Comment:
minor: I wouldn't do any treatment on the prefix (removing last "."), people should type exactly what they want there or else the behavior should be documented clearly to the user